sdk/src/androidTest/java/com/karhoo/sdk/api/LoyaltyIntegrationTest.kt

```diff
@@ -0,0 +1,230 @@
+package com.karhoo.sdk.api
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.tomakehurst.wiremock.junit.WireMockRule
+import org.assertj.core.api.Java6Assertions.assertThat
+import com.karhoo.sdk.api.model.LoyaltyBalance
+import com.karhoo.sdk.api.network.response.Resource
+import com.karhoo.sdk.api.testrunner.SDKTestConfig
+import com.karhoo.sdk.api.util.ServerRobot
+import com.karhoo.sdk.api.util.ServerRobot.Companion.EMPTY
+import com.karhoo.sdk.api.util.ServerRobot.Companion.GENERAL_ERROR
+import com.karhoo.sdk.api.util.ServerRobot.Companion.INVALID_JSON
+import com.karhoo.sdk.api.util.ServerRobot.Companion.LOYALTY_BALANCE
+import com.karhoo.sdk.api.util.ServerRobot.Companion.NO_BODY
+import com.karhoo.sdk.api.util.serverRobot
+import org.junit.After
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.net.HttpURLConnection
+import java.net.HttpURLConnection.HTTP_OK
+import java.net.HttpURLConnection.HTTP_UNAUTHORIZED
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+@RunWith(AndroidJUnit4::class)
+@Ignore("Tests not ready")
+class LoyaltyIntegrationTest {
+
+    @get:Rule
+    var wireMockRule = WireMockRule(SDKTestConfig.wireMockOptions)
+
+    private val latch = CountDownLatch(1)
+
+    @Before
+    fun setUp() {
+        serverRobot {
+            successfulToken()
+        }
+    }
+
+    @After
+    fun tearDown() {
+        wireMockRule.resetAll()
+    }
+
+    /**
+     * Given:   Loyalty Balance on a trip is requested
+     * When:    It is a user booking
+     * And:     Successful response has been returned
+     * Then:    The response payload should be valid
+     **/
+    @Test
+    fun loyaltyBalanceSuccess() {
+        serverRobot {
+            getLoyaltyBalanceResponse(code = HTTP_OK, response = LOYALTY_BALANCE)
+        }
+        var result: LoyaltyBalance? = LoyaltyBalance(123, false)
+
+        KarhooApi.loyaltyService.getBalance(ServerRobot.LOYALTY_ID).execute {
```

Comment:
Static imports

Reply:
Not sure what is being referred to here exactly, but again, Android studio newest version does not like it when I try to remove some static imports in the project.